content/id/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Kelola Sertifikat TLS Pada Klaster
+content_template: templates/task
+---
+
+{{% capture overview %}}
```

Comment:
This is out of date since https://github.com/kubernetes/website/pull/21359

Reply:
Thanks. I've rebased and update it as well.